pkgs/by-name/lo/lockbook/package.nix

```
@@ -24,7 +29,16 @@ rustPlatform.buildRustPackage rec {
     "lockbook"
   ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [
+    installShellFiles
+  ]
+  ++ lib.optionals isLinux [ makeWrapper ];
```

```suggestion
  ++ lib.optionals isLinux [ makeBinaryWrapper ];
```
Probably don't need the shell wrapper here.

---

Sorry about that I squashed incorrectly and undid all the review comments. I fixed it now, however.